clerk/app.py

```diff
@@ -12,44 +17,108 @@
 
 
 def main() -> int:
-    # app configuration
+    try:
+        eps = importlib.metadata.entry_points()["clerk.extensions"]
```

Comment:
Why limit to clerk extensions? What if I want to run ` hyperthymstic` on `JOURNAL_SAVED`?

Reply:
It's probably best to write a custom `clerk-hyperthymestic` plugin to achieve this - using the default `hyperthymestic` (or the default entrypoint of any other utility) is dangerous, since the hook callback function is liable to overwrite the entire journal with whatever it returns

We could update `hyperthymestic` to [return something other than `0`](https://github.com/josephhaaga/hyperthymestic/blob/main/hyperthymestic/console.py#L79), but then the design decisions of `clerk`'s plugin system propagate out into the design of all other applications

So the ideal solve is:
1. create a `clerk-hyperthymestic` that `install_requires` `hyperthymistic`
2. make it's `entry_points` command a function that uses `hyperthymestic` and determines what changes [if any] should be written to the journal, and returns a nice clean updated representation (`List[str]`) of the journal (or `None`) to be written